x-pack/solutions/security/plugins/security_solution/public/overview/components/host_overview/index.tsx

```diff
@@ -85,7 +85,7 @@ export const HostOverview = React.memo<HostSummaryProps>(
   }) => {
     const capabilities = useMlCapabilities();
     const userPermissions = hasMlUserPermissions(capabilities);
-    const darkMode = useDarkMode();
+    const darkMode = useKibanaIsDarkMode();
```

Comment:
nit: Would be nice to rename the boolean const to `isDarkMode`.

Reply:
Yeah I would like it to be consistent myself, but I just kept the naming as is, expect for `IS_DARK_MODE` which I think should be reserved for true constants. Feel free to change in a follow up PR.